### ISSUE TITLE
IDE1006 false positive in new C# project

### ISSUE THREAD

**Issue by vsfeedback** [dotnet/roslyn#43108]
_This issue has been moved from [a ticket on Developer Community](https://developercommunity.visualstudio.com/content/problem/975226/ide1006-false-positive-in-new-c-project.html)._

---
I added the following naming style convention for namespaces. Namespaces should be prefixed with "WeissGroup.WUT."
When I create the rule it works fine for the projects that exist in the solution.
However, if I create a new project this rule will issue IDE1006 for the new project regardless if the source code follows the convention or not. In other words I'm getting false positives for every namespace declaration!

Relevant contents of .editorconfig file:

```ini
dotnet_naming_rule.namespace_should_be_wutnamespace.severity = warning
dotnet_naming_rule.namespace_should_be_wutnamespace.symbols = namespace
dotnet_naming_rule.namespace_should_be_wutnamespace.style = wutnamespace

dotnet_naming_style.wutnamespace.required_prefix = WeissGroup.WUT.
dotnet_naming_style.wutnamespace.required_suffix = 
dotnet_naming_style.wutnamespace.word_separator = 
dotnet_naming_style.wutnamespace.capitalization = pascal_case
```

Class file in new project:

```csharp
namespace WeissGroup.WUT.ClassLibrary1    // IDE 1006 reported here even though correct 
{
    public class Class1
    {
    }
}
```

When I leave out the prefix then Visual Studio will even show tooltip and offer fixing of name violation. When I confirm to fix it then it changes to the correct name convention but still issues IDE 1006.

---
#### Visual Studio Feedback System on 4/2/2020, 08:04 PM: 

<p>We have directed your feedback to the appropriate engineering team for further evaluation. The team will review the feedback and notify you about the next steps.</p>

---
### Original Solutions
(no solutions)

**Comment by CyrusNajmabadi** [comment]
I don't believe dotted names are supported.

**Comment by CyrusNajmabadi** [comment]
Closing out as by design.  Naming rules are about a single name, not fully qualified names.  SO this is working as expected.  Haven't heard from otehrs for a feature request to expand hte design, so closing out.
